repo/packages/H/hello-world/0/command.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
The data-agility frameworks all use a python CLI (and therefore not .cli) because our tooling produces zip universe repos that don't work with .cli.  We'll be updating once the new packaging format is shipped.